index.js

```diff
@@ -60,7 +60,7 @@ function getAuthenticateHeader (authenticate) {
     }
   }
 
-  throw Error('Basic Auth: Invalid authenticate option')
+  next(new Error('Basic Auth: Invalid authenticate option'))
```

Comment:
Throwing error here will lead to `avvio` timeout as the throwing error is not properly consumed by `avvio`.
We need to pass the `Error` to the `callback` if we are using `callback-style` here.